model_analyzer/triton/client/http_client.py

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import gevent.ssl
```

Comment:
Do we need to do anything to make sure this is available in the container and/or wheel?

Reply:
@tanmayv25 is there a particular version or set of versions of [gevent](https://pypi.org/project/gevent/) that we need?

Reply:
i set it to require `gevent>=21.12.0` 👍